scripts/sct_label_vertebrae.py

```diff
@@ -323,14 +323,15 @@ def main(args=None):
             # automatically finds C2-C3 disc
             im_data = Image('data.nii')
             im_seg = Image('segmentation.nii')
-            im_label_c2c3 = detect_c2c3(im_data, im_seg, contrast)
+            im_label_c2c3 = detect_c2c3(im_data, im_seg, contrast, verbose=verbose)
```

Comment:
The parameter verbose is misleading, because here it is used for keeping temp files (not only to output terminal verbose). Therefore, I know it's not elegant, but i would do it that way:
```python
if not remove_temp_files:
  verbose_detect_c2c3 = 2
else:
  verbose_detect_c2c3 = 0
im_label_c2c3 = detect_c2c3(im_data, im_seg, contrast, verbose=verbose_detect_c2c3)
```